matching/ChangeLog.md

```diff
@@ -0,0 +1,3 @@
+# Changelog for haste
```

Comment:
I think we can delete this file.

Reply:
I agree on removing this for now. We can always add it later if we decide to make this a Haskell module, since it contains no useful information now.